spec/functions/strftime_spec.rb

```diff
@@ -1,9 +1,7 @@
 #! /usr/bin/env ruby -S rspec
 require 'spec_helper'
 
-describe "the strftime function" do
-  let(:scope) { PuppetlabsSpec::PuppetInternals.scope }
-
+describe 'strftime' do
   it "should exist" do
     expect(Puppet::Parser::Functions.function("strftime")).to eq("function_strftime")
```

Comment:
Harsh, @igalic , harsh. `strftime` needs to die a horrible death, while the `validate_ipv*` functions are here to stay and given their implementation, I do not trust them.